<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TEST</class>
 <widget class="QWidget" name="TEST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="modelfile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> 模型文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelfile_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="model_lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="image_lineEdit_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="meanfile_lineEdit_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modeldeploy">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>模型描述文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="image">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>测试图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="meanfile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>均值文件</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>10</y>
     <width>20</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="gpu">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>gpu_mode</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="cpu">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>cpu_mode</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="load">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="teststart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CNN可视化</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
